test/passenger_test.rb

```diff
@@ -69,6 +80,191 @@
+       expect(@passenger.net_expenditures()).must_equal 31.10
+    end 
+
+    it "sums multiple trips cost with incomplete" do
+    @costs = [44.55, 45.79]
+    @passenger = RideShare::Passenger.new(
+      id: 9,
+      name: "Merl Glover III",
+      phone_number: "1-602-620-2330 x3723",
+      trips: []
+      )
+    @trips = [
+      RideShare::Trip.new(
+        id: 8,
+        driver_id: 99,
+        passenger: @passenger,
+        start_time: start_time,
+        end_time: end_time,                             
+        rating: 5,
+        cost: @costs[0]
+        ),
+      RideShare::Trip.new(
+        id: 9,
+        driver_id: 99,
+        passenger: @passenger,
+        start_time: start_time,
+        end_time: end_time,
+        rating: 5,
+        cost: @costs[1]
+        )
+      ]
+      @trips.each do |trip|
+        @passenger.add_trip(trip)
+      end
+
+      expect(@passenger.net_expenditures).must_equal 90.34
+    end
   end
-end
+
+  describe "duration" do
+    it "sums one trip duration" do
+      @cost = 44.55
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+      )
+      trip = RideShare::Trip.new(
+        id: 8,
+        driver_id: 99,
+        passenger: @passenger,
+        start_time: start_time,
+        end_time: end_time,
+        rating: 5,
+        cost: @cost
+      )
+      @passenger.add_trip(trip)
+      expect(@passenger.total_time_spent).must_equal 86400
+    end
+
+    it "sums multiple trips duration" do
+      @costs = [44.55, 25.21]
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+      )
+      @trips = [
+        RideShare::Trip.new(
+          id: 8,
+          driver_id: 99,
+          passenger: @passenger,
+          start_time: Time.new(2016, 8, 8),
+          end_time: Time.new(2016, 8, 9),
+          rating: 5,
+          cost: @costs[0]
+        ),
+        RideShare::Trip.new(
+          id: 9,
+          driver_id: 99,
+          passenger: @passenger,
+          start_time: Time.new(2016, 8, 8),
+          end_time: Time.new(2016, 8, 9),
+          rating: 5,
+          cost: @costs[1]
+        )
+      ]
+      @trips.each do |trip|
+        @passenger.add_trip(trip)
+      end
+    end
+  end
+  
+  describe "total_time_spent" do
+    before do
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: [],
+      )
+      @trip1 = RideShare::Trip.new(
+        id: 8,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 9),
+        cost: 10,
+        rating: 5,
+      )
+      @passenger.add_trip(@trip1)
+    end
+  end
+
+      it "sums duration with incomplete trips" do
```

Comment:
This is an important edge case to test for. Nice work identifying it. However, I do not see where you have an incomplete trip. An incomplete trip should have an end_time of nil. In addition, you should include this it block in the `total_time_spent` describe block above.